go/pkg/qtag/declarations/icecap.yaml

```diff
@@ -0,0 +1,14 @@
+name: icecap
```

Comment:
why a new declared type? this appears as a nested doc in the design

Reply:
removed it